### ISSUE TITLE
`dephell project cd` command

### ISSUE THREAD

**Issue by orsinium** [dephell/dephell#70]
Store all projects (directories where dephell was runned with config) paths in the textfile in dephell's datadir and perform search for the best path by command

```bash
$ dephell project cd test
```

`~/local/dephell/paths.txt`:

```
/home/gram/Documents/lol/
/home/gram/Documents/test/
/home/gram/Documents/nani/
```

In this example dephell will cd into `/home/gram/Documents/test/`

**Comment by RPiAwesomeness** [comment]
I would like to take a shot at this for Hacktoberfest

**Comment by orsinium** [comment]
Great! Keep going :)

**Comment by RPiAwesomeness** [comment]
For clarification, not having used it before, does dephell currently store `paths.txt`? If not, how would you want it formatted/where should it be stored within the dephell data dir?

**Comment by orsinium** [comment]
Right, we have no it yet. Use, `TextCache('paths')`. It has `load` and `dump` methods that work with list of string. Exactly what you need :+1:
